mxcube3/ui/reducers/taskForm.js

```diff
@@ -1,15 +1,16 @@
 const initialState = {
-  sample_ids: [],
+  sampleIds: [],
```

Comment:
Ok. It's just a matter of agreeing. The Python standard is to have underscores, so I tend to write with underscores. But I don't mind having camel case in JS ; the only thing is that we should maybe try to add a rule if it exists in our linting tool, in order to see where we used bad names for variables in order to unify everything.